src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -186,7 +199,7 @@ create_distributed_table(PG_FUNCTION_ARGS)
 
 		ConvertToDistributedTable(relationId, distributionColumnName,
 								  distributionMethod, REPLICATION_MODEL_COORDINATOR,
-								  INVALID_COLOCATION_ID);
+								  INVALID_COLOCATION_ID, allowEmpty);
```

Comment:
These variable names are pretty baffling. Trying to think of a quick replacement to patch in before merge.